src/utils/number.tsx

```
@@ -1,4 +1,4 @@
-import { parseInt as parseIntLodash } from 'lodash';
+import { parseInt, parseInt as parseIntLodash } from 'lodash';
```

Biome (1.9.4)

[error] Do not shadow the global "parseInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)